test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/01_file_tailing_test.go

```diff
@@ -31,7 +32,8 @@ type LinuxFakeintakeSuite struct {
 //go:embed log-config/config.yaml
 var logConfig string
 
-var logPath = "/var/log/hello-world.log"
+var logFileName = "hello-world.log"
```

Comment:
```suggestion
const logFileName = "hello-world.log"
```
(probably the one below too)